engine/engine_test.go

```diff
@@ -382,40 +380,18 @@ func TestGetDefaultConfigurations(t *testing.T) {
 			}
 
 			defaultCfg, err := exchange.GetDefaultConfig(context.Background(), exch)
-			if err != nil {
-				t.Fatal(err)
-			}
-
-			if defaultCfg == nil {
-				t.Fatal("expected config")
-			}
-
-			if defaultCfg.Name == "" {
-				t.Error("name unset SetDefaults() not called")
-			}
-
-			if !defaultCfg.Enabled {
-				t.Error("expected enabled", defaultCfg.Name)
-			}
+			require.NoError(t, err, "GetDefaultConfig must not error")
+			require.NotNil(t, defaultCfg)
+			assert.NotEmpty(t, defaultCfg.Name)
+			assert.True(t, defaultCfg.Enabled)
```

Comment:
I think that when you're not testing the overall thing, you have to add context for the error.
That is: `require.NotEmpty(t, defaultCfg)` is fine, because it's about the unit test's target func main return value.
```suggestion
			assert.NotEmpty(t, defaultCfg.Name, "Name should not be empty")
			assert.True(t, defaultCfg.Enabled, "Enabled should have correct value")
```